tests/test_utils.py

```diff
@@ -341,6 +341,16 @@ def test_xz(self):
             path = get_release_file_path()
         self.assert_path('juju-gui-2.0.1.xz', path)
 
+    def test_tar_gz(self):
+        # The last release is correctly retrieved for tar.gz files too.
+        self.add('juju-gui-0.12.1.tgz')
+        self.add('juju-gui-1.2.3.tgz')
+        self.add('juju-gui-2.0.0+build.42.tgz')
+        self.add('jujugui-2.0.1.tar.gz')
+        with self.mock_releases_dir():
+            path = get_release_file_path()
+        self.assert_path('jujugui-2.0.1.tar.gz', path)
```

Comment:
The old way used .tgz.  The 'make dist' target of juju-gui now produces jujugui-VERSION.tar.gz.  I figured it was easier to adapt the charm to what was being produced than to either update juju-gui or require a manual renaming when copying to the charm.

I see no problem with keeping tgz and xz.